inc/class-wpseo-admin-bar-menu.php

```diff
@@ -184,6 +180,12 @@ protected function add_root_menu( WP_Admin_Bar $wp_admin_bar ) {
 			$alert_popup = $this->get_notification_alert_popup();
 		}
 
+		$score   = '';
```

Comment:
Moving this down will reset the _term_ score to empty string. Thus in the admin bar there's no score icon any longer for a term.

Icon on trunk:

<img width="283" alt="Screenshot 2019-10-09 at 09 46 31" src="https://user-images.githubusercontent.com/1682452/66461750-01494700-ea7a-11e9-8532-3b3f2d383765.png">

No icon on this branch:

<img width="278" alt="Screenshot 2019-10-09 at 09 46 39" src="https://user-images.githubusercontent.com/1682452/66461764-0ad2af00-ea7a-11e9-882e-e6c456fc2b3d.png">

Reply:
Good catch, refactoring.